content/en/contribute/medic/product-development-process/code-health.md

```diff
@@ -3,7 +3,7 @@ title: "Code Health"
 linkTitle: "Code Health"
 weight: 5
 description: >
-  How we keep our code and systems up-to-date
+  How to keep the code and systems up-to-date
```

Comment:
This isn't really how we are doing code health anymore... while Gareth is still owning the queue, there's not a dedicated person on code health each week. Focused Groups are responsible for code health in their area of concern and these are generally on OKRs as well. This means that we can do them when we want, but they gotta get done that quarter.

Reply:
I know, 😥 hence my comment [here](https://github.com/medic/cht-docs/pull/918#issuecomment-1381769429).
I was hoping that someone (like you 😊) from the reviewer list could suggest changes to the outdated pages with relevant updates. 
Any chance to suggest the relevant changes? 🙏

Reply:
Not today but I can do this section and the product trio activities one Thursday or Friday next week if nobody else gets to it before me!

Reply:
I have a [PR up for an update to code health process](https://github.com/medic/cht-docs/pull/939).